src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -615,4 +626,14 @@ public function stub($callback)
 
         return $this;
     }
+
+    /**
+     * Add the given middleware to the request.
+     */
```

Comment:
Please add the missing param and return phpdoc.

Reply:
Added these in the latest commit